recipes/sak/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+BINARY_HOME=$PREFIX/bin
+PKG_HOME=$PREFIX/opt/$PKG_NAME-$PKG_VERSION
```

Comment:
Is there any reason you put it under opt and not simply copy the binaries into PREFIX/bin?